app/features/recent-list/components/RecentList.js

```diff
@@ -64,14 +69,17 @@ class RecentList extends Component<Props, *> {
             <ConferenceCard
                 key = { conference.startTime }
                 onClick = { this._onNavigateToConference(conference) }>
-                <TruncatedText>
+                <ConferenceTitle>
```

Comment:
I wonder if the right thing to do should pass props? See [link](https://www.styled-components.com/docs/basics#passed-props)

`<TruncatedText bold = { true }> { conference.room } </TruncatedText>`

But I guess extending would be simpler?